public/javascript/pump/view.js

```diff
@@ -220,7 +220,7 @@
                         if (clientUrls.indexOf(name) !== -1) {
                             url += "-client";
                         }
-                        url += ".jade.js";
+                        url += ".pug.js";
```

Comment:
Nothing's wrong here but I want to a note to remind myself that admins _must_ rename from `.jade` to `.pug` due to this, even though technically serverside `pug` will load `.jade` files.